electron-app/magnifier/rust-sampler-manager.test.ts

```
@@ -0,0 +1,519 @@
+vi.mock('electron', () => ({
+  app: {
+    getAppPath: () => '/mock/app/path',
+  },
+}));
+
+vi.mock('electron-is-dev', () => ({
+  default: true,
+}));
+
+class MockChildProcess extends EventEmitter {
+  stdin: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)